calculators/memoria_rating.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+$GLOBALS['rating_calculators']['memoria_rating'] = [
+	'description' => '
+		<a href="/player/125980">Memoria\'s</a> rating, v1.0
+
+		Based on the main ratings.
+		- Starting rating is 2500 instead of 2000, minimum is capped at 100, and rating change ranges from 0 to 50, or 100 in case of being afk.
+		- Factor is changed from 4000 to 2000, meaning that at 4500 you have a 90% prediction to win against a 2500.
+		- Games under 3m40s are not counted towards rating.
+		- Only games from Standard and (Heroic) Commanders gamemodes are included.
+		- Team rating is based on the highest rated player and the lowest rated player.
```

Comment:
consider what's easier to read, list items with capital letter and dot in the end, or just lowercase sentences
![image](https://user-images.githubusercontent.com/1370909/233856820-e095867a-24fc-49e3-8040-6ac66803eaf4.png)
![image](https://user-images.githubusercontent.com/1370909/233856831-5a838675-6b52-4c2f-85b8-91540f8f6c2a.png)
up to you